template/dictionary_types.njk

```diff
@@ -17,6 +17,8 @@
 #ifndef GEN_DICTIONARY_{{name | snakecase | upper}}_H_
 #define GEN_DICTIONARY_{{name | snakecase | upper}}_H_
 
+using namespace::std;
```

Comment:
We just use std::string instead of this.
IDL string should be map with std::string.

Reply:
i fixed it to using std::string.